spec/models/user_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) { FactoryGirl.create(:user) }
+
+  it 'is valid with a name, email, and password' do
+    expect(user_1).to be_valid
+  end
+
+  it 'is invalid without a name' do
+    user_1.name = nil
+    user_1.valid?
```

Comment:
and again `user_1.valid?` without work. And here we could have an extra `expect`= `expect(user_1.valid?).to be()`